programs/voter-stake-registry/src/instructions/claim.rs

```diff
@@ -82,6 +88,16 @@ pub fn claim(ctx: Context<Claim>, realm_pubkey: Pubkey) -> Result<u64> {
     let vote_record =
         VoteRecordV2::deserialize(&mut &ctx.accounts.vote_record.data.borrow_mut()[..])?;
 
+    #[cfg(feature = "testing")]
+    require!(
+        realm_pubkey == Pubkey::from_str("DA5G7QQbFioZ6K33wQcH8fVdgFcnaDjLD7DLQkapZg5X").unwrap()
```

Comment:
can't we recreate the same DAO_PUBKEY in our tests?

Reply:
We can but this will be extremely inconvenient because this will bond us to the one instance at a time. To deploy a new staking on devnet we should update common-constants, then rebuild the program etc, that is tedious. It might be removed right after deploy, I'd say.

Reply:
No, `testing` flag is used for testing. However, you've suggested to change the `Pubkey` above to the `DAO_PUBKEY` constant from the `mpl-common-constant`, which is used in the devnet deployments.